### PR TITLE
update "integration test only" deps and move to devDependencies, closes #8600

### PR DESCRIPTION
**Pull request by lucidNTR**

### CODE REVIEW

**Review by alxndrsn — APPROVED** [review]
I wonder if `promise-polyfill` is still required - will check.

File: package.json [hunk 1]
```diff
@@ -111,6 +110,7 @@
     "throw-max-listeners-error": "1.0.1",
     "ua-parser-js": "0.7.24",
     "watch-glob": "0.1.3",
+    "whatwg-fetch": "3.6.2",
```
Comment (alxndrsn) [inline package.json:113]: @lucidNTR can you fix the order here?  I think `wd` should come before `whatwg-fetch`, and `npm` changes the order if I install/remove other deps.

**Comment by lucidNTR** [comment]
@alxndrsn the polyfill is only used at /tests/integration/index.html#L10 afaik, there seems to be an integration test testing pouch with ie and the polyfill.

**Comment by SourceR85** [comment]
Close, due to promise-polyfill and whatwg-fetch removed by #8825 and #8643.

**Pull request closed without merging**